src/frontends/pytorch/src/frontend.cpp

```diff
@@ -174,6 +175,7 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
     // the following 2 transformations are needed for keypoint detectron2 models to work.
     // AtenIndexToSelect will be called twice
     manager.register_pass<ov::pass::ConvertConvertLike>();
+    manager.register_pass<ov::pass::KeepFWPrecisionFor16BitFloatConstants>();
     manager.register_pass<ov::frontend::pytorch::pass::AtenIndexToSelect>();
 
     manager.register_pass<ov::pass::MarkDequantizationSubgraph>(
```

Comment:
This transformation does the same thing for int8 weights, can we put 16bit one near it and maybe make the similar naming? Or maybe even combine both of them in one transformation?

Reply:
Removed form onnx, pdpf, left in TF because CF is called there implicitly inside TransposeSinking.

Reply:
renamed and moved to in the vicinity of int8 marking